tests/test_porcelain.py

```diff
@@ -14,7 +14,7 @@ def test_list_all(tmpdir, runner, create):
     assert (
         result.output.strip() ==
         '{"completed": true, "due": 1451689200, "id": 1, "list": "default'
-        '", "percent": 26, "summary": "Do stuff", "urgent": false}'
+        '", "percent": 26, "summary": "Do stuff", "priority": None}'
```

Comment:
We can't remove urgent, nope. And I think it makes sense to add priority. We **can** show the numeric value here.